crates/api_models/src/payments.rs

```diff
@@ -203,7 +203,7 @@ pub struct PaymentsRequest {
 
     /// Business country of the merchant for this payment
     #[schema(example = "US")]
```

Comment:
add value_type = CountryAlpha2 here. The idea is to add this wherever the enum/struct is imported from a module, value_type has to be added.